### PR TITLE
Reduce time range on 'Show received messages' button

### LINKED ISSUE #5620: 'Show received messages' link can trigger OOM with field charts query
Once a chart is created on the search page, it keeps its initially configured resolution
and query. Extending the search time range to `all messages` can create a query
that runs either Graylog or Eleasticsearch out of memory.
(See #5559)

This is easily achieved by accident, since the 'Show received messages' link
on inputs defaults to `range=0`.

## Possible Solution
Change the link to use a range of `last 7 days` instead.

## Steps to Reproduce (for bugs)

chart-oom

* Graylog Version: 3.0 rc1
* Elasticsearch Version: 6.6

### PR DESCRIPTION
**Pull request by mpfz0r**
Defaulting to search in all messages is expensive and might trigger
OOM scenarios much easier.

Instead of searching all messages, we are searching only
messages in a range of past and future 26 hours.

26 hours because that's the largest possible timezone mistake a user can make.

Fixes #5620

### CODE REVIEW

**Comment by bernd** [comment]
Refs https://github.com/Graylog2/graylog2-server/issues/4533

**Comment by mpfz0r** [comment]
> I think @edmundoa's suggestion of using a drop-down to offer different values is a reasonable compromise between the possibility of killing you ES cluster and not finding your messages. slightly_smiling_face
> 
> [#4533 (comment)](https://github.com/Graylog2/graylog2-server/issues/4533#issuecomment-361553051)
> 
> What do you think?

From a discussion I had (forgot with whom) the reason why it was changed to "all messages" was that
this would show messages from the future. Apparently some of our users haven't discovered NTP yet..

So couldn't we just set the time range to 'last 7 days - now + 1 day' or so?

**Comment by bernd** [comment]
> So couldn't we just set the time range to 'last 7 days - now + 1 day' or so?

That could be done by using an absolute time range parameter instead of the relative one, yes.

File: graylog2-web-interface/src/util/TimeRangeHelper.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Creates an absolute time range that can be used to look up recently received messages.
+ * It accommodates for the eventuality that messages can have timestamps from the future
+ * due to wrong timezone parsing or wrong system clocks.
+ */
+const recentMessagesTimeRange = (): AbsoluteRangeQueryParameter => {
+  const now = Date.now();
+  // The biggest possible time difference on earth is 26 hours.
+  // It's between Kiribati is UTC+14 and the Howland Islands UTC-12
```
Comment (mpfz0r) [inline graylog2-web-interface/src/util/TimeRangeHelper.ts:28]:
```suggestion
  // It's between Kiribati (UTC+14) and the Howland Islands (UTC-12)
```

**Review by ousmaneo — APPROVED** [review]
Tested and works!

**Review by bernd — APPROVED** [review]

**Pull request merged**